src/biokbase/narrative/tests/test_jobmanager.py

```diff
@@ -40,13 +41,52 @@ def create_jm_message(r_type, job_id=None, data=None):
     return {"content": {"data": data}}
 
 
-def get_job_state(job_id, exclude_fields=None):
-    info = copy.deepcopy(test_jobs.get(job_id, {}))
-    if exclude_fields:
-        for f in exclude_fields:
-            if f in info:
-                del info[f]
-    return info
+def get_retry_job_state(orig_id, status="unmocked"):
+    return {
+        "state": {
+            "job_id": orig_id[::-1],
+            "status": status,
+            "batch_id": None,
+            "job_output": {},
+            "cell_id": None,
+            "run_id": None,
+            "child_jobs": [],
+        },
+        "widget_info": None,
+        "owner": None,
+    }
+
+
+def get_dne_job_state(job_id):
+    return {"state": {"job_id": job_id, "status": "does_not_exist"}}
+
+
+def get_test_job_states():
```

Comment:
This sort of simulates that combination of job._augment_ee2_state and job.revised_state